src/main/java/edu/cornell/kfs/pmw/batch/PaymentWorksKeyConstants.java

```diff
@@ -40,6 +40,7 @@
     public static final String NEW_VENDOR_DETAIL_WAS_NOT_FOUND_ERROR_MESSAGE = "error.paymentworks.new.vendor.detail.not.returned.by.web.service";
     public static final String DUPLICATE_NEW_VENDOR_REQUEST_ERROR_MESSAGE = "error.paymentworks.duplicate.new.vendor.request"; 
     public static final String INITIAL_SAVE_TO_PMW_STAGING_TABLE_FAILED_ERROR_MESSAGE = "error.paymentworks.intial.staging.table.save.failed";
+    public static final String INITIAL_PAYMENT_WORKS_VENOR_SAVE_ERROR_MESSAGE = "error.paymentworks.paymentworksvendor.initial.save";
```

Comment:
Minor typo in name of constant.

Reply:
thanks, I will change

Reply:
with removing the report data elements mentioned below, this can be removed entirely